doc/install.rst

```diff
@@ -16,173 +19,7 @@ Installation
    (both Raspbian Jessie and Stretch) models A, B2, 3B, Zero, Zero W and
```

Comment:
Maybe leave out the distro names, since we have to update it with every release.

Reply:
Since it references specific work that you have done, perhaps it should remain as is.  I would suggest eventually including a dedicated page on test results.